### PR TITLE
Add CloudFolder.md

### PR DESCRIPTION
**Pull request by nelson0217**

### CODE REVIEW

**Comment by dineshshenoy** [comment]
I couldn't locate your form entry (https://slingshot.filecoin.io/register-now).  Could you submit this form to complete the registration process?

**Comment by nelson0217** [comment]
I have submitted this form. My github handle is [filcloud](https://github.com/filcloud).

**Comment by nelson0217** [comment]
If we use Textile Hub, how to get the Powergate address?

File: participants/CloudFolder.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+At least 1 TiB, there may be a lot more with the increase of users.
+
+## (7) How are you structuring the data?
+
+We will push the data to IPFS/Filecoin when users upload it.
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+Encrypt, and then push to IPFS/Filecoin. We plan to associate indexes with users and store them in the database, if there is no better way.
```
Comment (dineshshenoy) [inline participants/CloudFolder.md:42]: Please provide a way to decrypt so reviewers can validate the data on Filecoin.  Thank you.

**Review by dineshshenoy — APPROVED** [review]

**Pull request merged**